### ISSUE TITLE
ExecPath not working

### ISSUE THREAD

**Issue by 0xrishabh** [chromedp/chromedp#661]
ExecPath is only working if the path provided in is $PATH and the name of the binary is in the predefined array of binary name list.

is it suppose to be this way?

**Comment by mvdan** [comment]
You can pass it an absolute path, which it will use directly. I'm not sure what you mean. This is very well documented:

> The given path can be an absolute path to a binary, or just the name of the program to find via exec.LookPath.

**Comment by 0xrishabh** [comment]
I am giving it an absolute path but it's still not working.
Can you give me an example code that uses ExecPath and is working. I did not find any in here

**Comment by 0xrishabh** [comment]
I want to correct my self -->

/random/random/chromium works but
/random/random/change_chromium_to_weird_name
does not work as path in ExecPath

Is this how it's suppose to work or is it bug?

**Comment by poor-branson** [comment]
How about this

Lets say there is an chromium executable package in the same folder of your app, like this
| workspace
--| main.go
--|Google Chrome/chrome

Then you can use it like this

```go
dir, dirError := os.Getwd()
if dirError != nil {
	panic(dirError)
}
opt := []func(allocator *chromedp.ExecAllocator){
	chromedp.ExecPath(path.Join(dir, "Google Chrome", "chrome")),
}

allocatorCtx, allocatorCancel := chromedp.NewExecAllocator(
	context.Background(),
	append(opt, chromedp.DefaultExecAllocatorOptions[:]...)[:]...,
)

ctx, err := chromedp.NewContext(allocatorCtx)
chromedp.Run(ctx, ...)
```

**Comment by ZekeLu** [comment]
> /random/random/change_chromium_to_weird_name

It's not cleared what's in the _weird_name_, but I have tested the following names on Ubuntu, all works:

* space in name: `chrome a`
* `--` in name:  `chrome --a`
* emoji: `🌺`

> error: unknown flag 'disable-background-networking'

**flag** is special in golang. So I guess `/random/random/change_chromium_to_weird_name` is a path to an executable compiled from go source. I compiled an executable from the following source, and observed the similar error (`flag provided but not defined: -foo`, `foo` changes  every time).  

```go
// go 1.16.3
package main

import (
	"flag"
)

func main() {
	flag.Parse()
}
```

Anyway, it's very rare that one will change the browser to a _weird_name_, so closing this issue.
